walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -4,6 +4,14 @@
   width: 650px;
 }
 
+#header{
+    position: relative;
+
+}
+p#me img {
+background-position-x: 50px; /*move it to the left*/
```

Comment:
Add a space in the comment so the text isn't next to the comment markers 

`/*[space]move it to the left[space]*/`